.idea/workspace.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we add this file to .gitignore? I think this is just VSCode's autogenerated config.